templates/nextflow-tower-s3-bucket.j2

```diff
@@ -0,0 +1,191 @@
+AWSTemplateFormatVersion: '2010-09-09'
```

Comment:
this template is copied from https://github.com/Sage-Bionetworks/s3-synapse-sync/blob/master/s3-synapse-sync-bucket.j2  only the lambda portion was removed since that is specific to s3-synapse-sync.